docs/deployment/connect-scm.md

```diff
@@ -60,7 +77,23 @@ You have successfully connected an org in Semgrep AppSec Platform with an organi
 
 For users of GitLab cloud-hosted plans, a connection to GitLab is created automatically after [adding a Semgrep job to GitLab CI/CD](/deployment/add-semgrep-to-ci). No other steps are needed.
 
-## Connect to on-premise orgs
+## Connect to on-premise orgs and projects
+
+### Bitbucket Data Center
+
+1. Sign in to Semgrep AppSec Platform.
+1. Go to **Settings** > **Source Code Managers**, and click **Add Bitbucket Data Center**.
+![Source code manager tab](/img/source-code-manager.png#md-width)
+1. In the **Connect your Bitbucket Project (Key)** dialog box, provide:
```

Comment:
Minor tidbit that is up to you: when the UI of something you're documenting jumps between ALL CAPS or Sentence case or Title Case, you can make it sentence case. (Those guys at google knew this happens)
![image](https://github.com/user-attachments/assets/d2b87bd3-18a7-4a37-848f-e1b2e1e002cd)


In this case, it could be Connect your Bitbucket Project (key)?